src/servant/tt_002/faerie_data_2C24.c

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "faerie.h"
+
+s16 D_us_80172C24[] = {0x0000, 0x0022, 0x0472, 0xFFFF, 0x000E, 0x0000};
+
+s16 D_us_80172C30[] = {0x0000, 0x0022, 0x0473, 0xFFFF, 0x000E, 0x0000};
+
+// This is a ranked lookup table.  First column is selected from a rand() %
+// 0x100 to select the data at the pointer in column 2
+s32 D_us_80172C3C[] = {0x0000007F, D_us_80172C24, 0x000000FF, D_us_80172C30};
```

Comment:
That's a good point.  How do we fix that if the main psx and psp builds need an array of s32?  Do we need to have it switch between s32 and s64 based on build?
```suggestion
#ifdef PLATFORM_64BIT
s64 D_us_80172C3C[] = {0x0000007F, D_us_80172C24, 0x000000FF, D_us_80172C30};
#else
s32 D_us_80172C3C[] = {0x0000007F, D_us_80172C24, 0x000000FF, D_us_80172C30};
#endif
```

Reply:
I did try to get a struct put together, but was really unable to do so and get it to match.  I'll noodle this for a bit and after I get the data imported and the variables documented/renamed I'll figure something out when adding tt_002 to the CMake lists.

Reply:
I don't think it's going to match with a struct, the i * 2 pattern is all over the codebase, they just used flat arrays a lot. It would have to be in an ifdef